microsetta_private_api/repo/admin_repo.py

```diff
@@ -553,7 +553,7 @@ def get_project_barcodes(self, project_id):
                 cur.execute(query, [project_id, ])
                 return list([v[0] for v in cur.fetchall()])
 
-    def get_barcodes_filter(self, kit_ids=None, emails=None,
+    def get_barcodes_filter(self, kit_ids=None, box_ids=None, emails=None,
                             outbound_tracking_numbers=None,
                             inbound_tracking_numbers=None,
                             dak_order_ids=None):
```

Comment:
Adding to the end out of caution in the event this method is called somewhere without relying on kwargs

```suggestion
    def get_barcodes_filter(self, kit_ids=None, emails=None,
                            outbound_tracking_numbers=None,
                            inbound_tracking_numbers=None,
                            dak_order_ids=None, box_ids=None):
```